crates/neon-runtime/src/napi/error.rs

```diff
@@ -1,11 +1,23 @@
-use raw::Local;
+use raw::{Env, Local};
 
-pub unsafe extern "C" fn throw(_val: Local) { unimplemented!() }
+use nodejs_sys as napi;
 
-pub unsafe extern "C" fn new_error(_out: &mut Local, _msg: Local) { unimplemented!() }
+pub unsafe extern "C" fn throw(env:Env,error: Local)->bool {
+    let status=napi::napi_throw(env,error);
+    status==napi::napi_status::napi_ok
+}
 
-pub unsafe extern "C" fn new_type_error(_out: &mut Local, _msg: Local) { unimplemented!() }
+pub unsafe extern "C" fn new_error(out: &mut Local,env:Env,code:Local,msg:Local)->bool {
```

Comment:
#Fix: Please follow formatting of other functions.

Reply:
fixed